src/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+.h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin-top: 0;
+}
+
+h2 {
```

Comment:
do not style tag, create a class for it. except `body`

Reply:
I fixed, thanks